### PR TITLE
feat: minimal dashboard page

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: https://github.com/lightdash/lightdash/issues/4599

### Description:

CleanShot 2023-03-03 at 20 09 13@2x

### CODE REVIEW

File: packages/frontend/src/components/DashboardTiles/TileBase/TileBase.styles.tsx [hunk 1]
```diff
@@ -2,12 +2,8 @@ import { Card, Colors, H5 } from '@blueprintjs/core';
 import styled, { createGlobalStyle } from 'styled-components';
 
 interface HeaderContainerProps {
-    isEditMode: boolean;
-    isHovering?: boolean;
-}
-
-interface TitleWrapperProps {
-    hasDescription: boolean;
+    $isEditMode: boolean;
```
Comment (rephus) [inline packages/frontend/src/components/DashboardTiles/TileBase/TileBase.styles.tsx:5]: why did you use this $?

**Review by rephus — APPROVED** [review]
I tested it and it works great ! :raised_hands: 

Just 1 thing, should we show an error if the dashboard does not exist or there is an error ? I get a blank page here : 

https://lightdash-pr-4674.onrender.com/minimal/projects/3675b69e-8324-4110-bdca-059031aa8da3/dashboards/invalid-uuid

File: packages/frontend/src/components/DashboardTiles/TileBase/TileBase.styles.tsx (hunk 1 above)
Comment (IrakliJani) [reply packages/frontend/src/components/DashboardTiles/TileBase/TileBase.styles.tsx:5]: thanks for the review @rephus!!

nothing major but if you have a prop with `$` prefix it does not pass that prop to the DOM node thus it does not log bunch of warnings
https://styled-components.com/docs/api#transient-props

**Comment by IrakliJani** [comment]
great catch @rephus, I just pushed a fix for that for both minimal dashboard and chart as well

CleanShot 2023-03-06 at 12 02 07@2x
CleanShot 2023-03-06 at 12 01 48@2x

**Review by rephus — APPROVED** [review]

**Pull request merged**